components/ai_chat/resources/page/utils/file_utils.ts

```diff
@@ -19,11 +19,17 @@ export const convertFileToUploadedFile = (
       }
 
       const uint8Array = new Uint8Array(arrayBuffer)
+
+      // Determine file type based on MIME type
+      const fileType = file.type.toLowerCase() === 'application/pdf'
+        ? Mojom.UploadedFileType.kPdf
+        : Mojom.UploadedFileType.kImage
```

Comment:
can we error out instead of fallback to image if it is type we can't handle?

Reply:
Addressed in the latest